### ISSUE TITLE
Autoscroll happening when dragging & clicking on application in system without stopping,

### ISSUE THREAD

**Issue by manojmanchukonda** [atlassian/pragmatic-drag-and-drop#158]
https://github.com/user-attachments/assets/3f1b952b-ab65-4a56-9d8a-0c9b4057fd94

When dragging an item in the scroll container & clicking on any opened in the menu bar at the bottom without releasing the drag action, the container is autoscrolling and the whole browser is autoscrolling without any interaction, is there any way to stop when we click on any system application?

**Comment by alexreardon** [comment]
I am not _totally_ following your question.

Would you like the auto scroller to stop scrolling after the user has left the `window`?

**Comment by manojmanchukonda** [comment]
Yes, If we are dragging an item using the mouse and clicks pointer outside of the browser window without releasing the drag action then we need to stop autoscroll
